cpp/ql/lib/experimental/semmle/code/cpp/semantic/analysis/SimpleRangeAnalysisWrapper.qll

```diff
@@ -0,0 +1,25 @@
+import cpp
```

Comment:
Nothing from this file is being used elsewhere yet. Do I see that correctly?

Reply:
Oh, didn't mean to include that here.